tests/checkers/rstcheck.py

```diff
@@ -1,63 +1,34 @@
-"""Sanity test using rstcheck and sphinx."""
-from __future__ import annotations
-
-import re
-import subprocess
+from rstcheck_core.runner import RstcheckMainRunner
+from rstcheck_core.config import RstcheckConfig
+import pathlib
 import sys
 
-
 def main():
-    paths = sys.argv[1:] or sys.stdin.read().splitlines()
-
-    encoding = 'utf-8'
-
-    ignore_substitutions = (
-        'br',
+    # Define the paths to check (passed as CLI arguments or from stdin)
+    paths = [pathlib.Path(p) for p in (sys.argv[1:] or sys.stdin.read().splitlines())]
```

Comment:
This expression is unnecessarily complex. Split it.

Reply:
Done